src/Shared/Configuration/Files/World.cs

```diff
@@ -42,6 +42,11 @@ public class WorldConfFile : ConfFile
 		public int JobMaxRank { get; protected set; }
 
 		// skills.conf
+		public float NormalAttackMultiplier { get; protected set; }
+		public float PlayerSkillMultiplier { get; protected set; }
+		public float EnemySkillMultiplier { get; protected set; }
```

Comment:
That's true.  I've renamed them to be "Monster".  Incidentally, I wonder if this affects Summoner Summons?  I kind of hope not, it would almost certainly make them far too powerful (if they are affect they may need a specific exception).  We'll deal with that when we implement Summoner I guess.

Reply:
Since everything that's not a player is a monster, summons are almost certainly monsters as well. As such, one might also wonder if there should be even more multipliers to factor them in. Though having a dozen fine-grained multipliers somewhat feels overkill as well^^"